INFRASTRUCTURE.md

```diff
@@ -0,0 +1,62 @@
+# Our Infrastructure
+
+First, let's address some details of this document.
+
+<details>
+  <summary>What, why, who...</summary>
+
+  ## What?
+
+  This document outlines the various systems that JSON Schema the organisation uses and maintains.
+  This does not include social media profiles.
+
+  ## Why?
+
+  It's part of the required tasks to be completed by the OpenJS Foundation, forming an item of the [onboarding checklist](https://github.com/openjs-foundation/cross-project-council/blob/main/PROJECT_PROGRESSION.md#onboarding-checklist).
+  It's also probably good practice.
+
+  This enables members of the JSON Schema team, and the OpenJS Foundation, to have oversight and management of such infrastructure should it be required.
+
+  ## Who?
+
+  This was mainly a list created by Ben Hutton (@relequestual).
+  Others are free to amend and update this (Via Pull Requests) as is required.
+
+</details>
+
+JSON Schema mostly operates on GitHub and Slack.
+
+Our GitHub organisation is located at https://github.com/orgs/json-schema-org
+
+Our Slack server is located at https://json-schema.slack.com
+(Anyone can join using https://json-schema.org/slack)
+
+The JSON Schema organisation maintains ONE domain name: json-schema.org.
+
+The domain name is registered and paid for by the OpenJS Foundation.
+
+The domain name defers its name servers to CloudFlare.
+The DNS mostly points to GitHub.
+
+The website is split into several applications for historical reasons.
+
+The main website is run through GitHub Pages. The source code is located at https://github.com/json-schema-org/json-schema-org.github.io
+
+The human friendly documentation for the specification, dubbed "Understanding JSON Schema", is located at https://json-schema.org/understanding-json-schema
+The source for this project is located at https://github.com/json-schema-org/understanding-json-schema
+
+Both of these sites are deployed by GitHub onto GitHub Pages.
+
+The JSON Schema blog is located at https://json-schema.org/blog
+The source code is located at https://github.com/json-schema-org/blog
+
+The blog is deployed on CloudFlare Pages.
```

Comment:
Any reason why the website cannot be on GitHub Pages? That would unify both the site and the blog, and decrease the amount of things to be worrying about in terms of infrastructure.

Reply:
That would take someone some effort to do. The blog has a url rewrite to make it run from `/blog`.
I figured it wasn't worth the effort when work is underway to merge the THREE sites together under one roof.
If we deploy to cloudflare, we can get some nice bonuses, more control, and better stats, than hosting on GitHub.